src/Controls/src/Core/Platform/Android/SwipeGestureHandler.cs

```diff
@@ -36,34 +38,65 @@ public bool OnSwipe(float x, float y)
 			if (view == null)
 				return false;
 
+			var transformedCoords = TransformSwipeCoordinatesWithRotation(x, y, view.Rotation);
+
+			_totalX += PixelTranslation(transformedCoords.x);
+			_totalY += PixelTranslation(transformedCoords.y);
+
 			var result = false;
 			foreach (SwipeGestureRecognizer swipeGesture in
-					 view.GestureRecognizers.GetGesturesFor<SwipeGestureRecognizer>())
+					view.GestureRecognizers.GetGesturesFor<SwipeGestureRecognizer>())
 			{
-				((ISwipeGestureController)swipeGesture).SendSwipe(view, PixelTranslation(x), PixelTranslation(y));
+				((ISwipeGestureController)swipeGesture).SendSwipe(view, _totalX, _totalY);
 				result = true;
 			}
 
 			return result;
 		}
 
+		(float x, float y) TransformSwipeCoordinatesWithRotation(float x, float y, double rotation)
+		{
+			var correctedX = x; 
+			var correctedY = y;  
```

Comment:
Never are modified, cn use directly x and y.

Reply:
I have updated the code to use x and y directly since they are not modified.